pegasus/sites.v3/code.org/views/homepage_hero.haml

```diff
@@ -58,8 +58,17 @@
         %a.herohocbutton.herolink-white{href: entry[:href]}
           %i.fa{class: entry[:icon_class]}
           %span= entry[:plain_text]
-
-
+      - elsif entry[:type] == "flex-container"
+        %div.flex-container
+          .text-wrapper
+            %h1
+              = I18n.t(entry[:text_h1])
+            %p
+              = I18n.t(entry[:text_desc])
+            %a.link-button{href: entry[:button_link]}
+              = I18n.t(entry[:button_text])
+          %figure
```

Comment:
Out of curiosity, what's the benefit of using the `figure` tag?

Reply:
It's better for accessibility, and I find it useful for styling since it acts as a semantic container for images (and videos) — here's a bit more about it: [https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figure](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figure)